rules/nod-vehicles.yaml

```diff
@@ -1351,3 +1240,215 @@ WEED:
 		LocalOffset: 543,0,0
 		Palette: effect
 	HitShape:
+
+SCORPION:
+	Inherits@1: ^Walker
+	Inherits@2: ^SuperHeavyVehicle
+	Inherits@FeatureAutotargetVehicle: ^FeatureAutotargetVehicle1
+	Inherits@FeatureAutotargetInfantry: ^FeatureAutotargetInfantry2
+	Inherits@FeatureAutotargetBuilding: ^FeatureAutotargetBuilding3
+	Valued:
+		Cost: 8000
+	Tooltip:
+		Name: Nod Scorpion
+	Selectable:
+		Bounds: 42,82, 0, -20
+	Buildable:
+		Queue: Vehicle.Nod
+		BuildPaletteOrder: 254
+		Prerequisites: ~naweap, napyra, ~!t1, ~!t2, ~technology
+		BuildLimit: 1
+		Description: Heavy Walker armed with a strong laser and also is cloaked.\n\nGood vs: Ground\n\nSpecial:\n- This unit is cloaked\n- This unit can emp\n- Crush class: Super Heavy vehicle
+	Voiced:
+		VoiceSet: Avatar
+		Volume: 2
+	Mobile:
+		TurnSpeed: 5
+		Speed: 65
+	GrantConditionOnMovement@steps:
+		Condition: walker
+	AmbientSound@step1:
+		SoundFiles: stomp2a.aud, stomp2b.aud, stomp2c.aud
+		Delay: 3
+		Interval: 21
+		RequiresCondition: walker
+	Health:
+		HP: 5000
+	SelfHealing:
+		Step: 1
+		PercentageStep: 0
+		Delay: 3
+		HealIfBelow: 1
+		DamageCooldown: 200
+	RevealsShroud:
+		Range: 12c745
+	BodyOrientation:
+		QuantizedFacings: 8
+	AttackFrontal:
+	Armament@primary:
+		Weapon: ScorpionLaser
+		Sequence: shoot
+		LocalOffset: 750,0,2000
+		MuzzlePalette: apolra50alpha
+		RequiresCondition: !rank-heroic
+	Armament@eyes:
+		Weapon: RaiderGunLaser
+		LocalOffset: 924,350,654
+	Armament@eyes2:
+		Weapon: RaiderGunLaser
+		LocalOffset: 924,-350,654
+	WithInfantryBody:
+		DefaultAttackSequence: shoot1
+	WithDeathAnimation:
+		DeathTypes:
+			BulletDeath: 1
+			SmallExplosionDeath: 1
+			ExplosionDeath: 1
+			FireDeath: 1
+			EnergyDeath: 1
+	RenderSprites:
+		Scale: 1.0
+	WithMuzzleOverlay:
+	HitShape:
+	Cloak@self:
+		InitialDelay: 90
+		CloakDelay: 90
+		CloakSound: stealth01.aud
+		UncloakSound: unstealth01.aud
+		IsPlayerPalette: true
+		UncloakOn: Attack, Unload, Infiltrate, Demolish, Damage
+	Explodes:
+		Weapon: UnitExplodeEpic
+		EmptyWeapon: UnitExplodeEpic
+	ActorLostNotification:
+		Notification: CriticalUnitLost
+
+MONTAUK:
```

Comment:
sure you even want this ingame? I understand this is an epic, but it has subterrenean + bansheecannon (easily nuking weapon) combo, which is kinda gamebreaking if somebody at least knows how to micro by tiny bit, if it wasn't subterrenean it might've worked normally